www/source/docs/create-packages-binary-only.html.md

```diff
@@ -10,22 +10,14 @@ You can write plans to package up these binary artifacts with minimal special ha
 
 ## Override The Phases You Don't Need
 
-A Habitat package build proceeds in phases: download, verification, unpacking (where you would also patch source code, if you had it), build, and finally installation. You can override the behavior of any of these phases by redefining the corresponding `do_` function. The following is an extreme example of overriding all the phases except install:
+A Habitat package build proceeds in phases: download, verification, unpacking (where you would also patch source code, if you had it), build, and finally installation. Each of these phases has default behavior within the build system.
```

Comment:
We should link to http://localhost:4567/docs/reference/plan-syntax/#callbacks here.